File: leetcode/1791-richest-customer-wealth/richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int result = 0;

        for (const auto& row : accounts) {  // auto& == vector<int>
            int sum = 0;

            for (int column : row) {
                sum += column;
            }
            result = max(result, sum);
        }

        return result;
    }
};

File: leetcode/1791-richest-customer-wealth/richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        result = 0

        for costumerAccounts in accounts:
            costumerSum = sum( costumerAccounts )

            if costumerSum > result:
                result = costumerSum
        
        return result
        